sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/jul/ConsoleHandler.java

```diff
@@ -42,6 +47,9 @@ class ConsoleHandler extends Handler {
      * A Handler which publishes log records to System.err.
      */
     private StreamHandler stderrHandler;
+    @SuppressWarnings("PMD.ThreadPoolCreationRule")
```

Comment:
You have to follow the Java code guidelines rather than skip it with `@SuppressWarnings`. The queue should be bounded.

Reply:
has deleted